api/webhook-vs-polling.md

```diff
@@ -93,20 +92,20 @@ Con's:
 
 * Not real time, a polling period is typically 5 mins.
 * You need to build or configure a periodic polling pattern at your side.
-* You cannot filter on which order events to receive, you will receive any order line change.
-* You cannot see what order event happened.
+* You cannot filter on which order or shipments events to receive, you will receive any order line change.
+* You cannot see what order or shipment event happened, multiple events may have happened.
 {% endhint %}
 
 ### Polling usage
 
-#### Step 1. Fetch updated orders periodically
+#### Step 1. Fetch updated orders or shipments periodically
 
-Fetch every polling period, typically 5 minutes, all orders which are new or changed since last date time.
+Fetch every polling period, typically 5 minutes, all orders or shipments which are new or changed since last date time.
 
 * Use the latest `lastUpdatedAt` from previous poll request in the `lastUpdatedSince` filter.
 * Sorting is set automatically to `lastUpdatedAt` order `asc` \(latest `lastUpdatedAt` will be in the last order in the response body\)
```

Comment:
I think the sentence below fitst better. 
(latest `lastUpdatedAt` will be last in the response body\

Reply:
Ah, I misread the `order` part. But `latest lastUpdatedAt will be` reallly `in the last order in the response body`
What means `will be last in the response body`?

Reply:
improved